templates/docs/examples/patterns/image/container/highlighted.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Image container / Highlighted{% endblock %}
```

Comment:
A bit of inconsistent naming:

```suggestion
{% block title %}Image Container / Highlighted{% endblock %}
```